### PR TITLE
Mira stratify drilldown update

### LINKED ISSUE #2357: [FEAT]: TA2 Operator Stratify model
### Objective
Each version will be developed and managed within its own Pull Request (PR).

### Version 1
Allow only one stratification per operator in the _Wizard_ view.

Screenshot 2023-12-04 at 11 23 29 AM

### Version 2
Allow only one stratification per operator in the _Notebook_ view.

Screenshot 2023-12-04 at 11 25 48 AM

### Version 3
Add tables of _variables_ and _parameters_ underneath the model diagram in the _preview_ section

Screenshot 2023-12-04 at 11 27 11 AM

### Version 4
Add drilldown on matrices to see rate equations.

### PR DESCRIPTION
**Pull request by Jami159**
Resolves #2357

https://github.com/DARPA-ASKEM/terarium/assets/10509899/56708531-80b8-4034-8036-4fcd2c740a97

### CODE REVIEW

File: packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue [hunk 1]
```diff
@@ -101,22 +104,26 @@ watch(
...
+.input-row {
+	width: 100%;
 	display: flex;
-	flex-direction: column;
-	padding-right: 2rem;
-	padding-bottom: 1rem;
+	flex-direction: row;
+	flex-wrap: wrap;
+	align-items: center;
+	gap: 0.5rem;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue:117]: You can use `var(--gap-small)` for this!
https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L950
```suggestion
	gap: var(--gap-small);
```

File: packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue [hunk 2]
```diff
@@ -101,22 +104,26 @@ watch(
...
 
-.select-variables {
+.label-and-input {
 	display: flex;
 	flex-direction: column;
-}
-
-.subdued-text {
-	color: var(--text-color-subdued);
+	gap: 0.5rem;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue:127]:
```suggestion
	gap: var(--gap-small);
```

**Review by YohannParis — APPROVED** [review]
Great Job 🎉 
Please share that video in #askem-terarium, people would love to see it, it's awesome!!!

File: packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue (hunk 1 above)
Comment (Jami159) [reply packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue]: done!

**Pull request merged**